### ISSUE TITLE
Nomad must be run as root for 'exec' driver to work

### ISSUE THREAD

**Issue by lanzafame** [libp2p/testlab#9]
According to [Nomad docs](https://www.nomadproject.io/docs/drivers/exec.html#client-requirements) running `nomad agent -dev` is not sufficient, it must be run as root.

Related to #5.

**Comment by lanzafame** [comment]
@bigs ok so the vm route for running testlab is ideal regardless if you are running linux on host or not due Nomad running out of disk space when tries to copy in the universe: https://www.nomadproject.io/docs/drivers/exec.html#chroot

**Comment by bigs** [comment]
@lanzafame yeah, that is a big issue when running nomad in development. in production, i expect cleaner filesystems, which makes it preferable to the docker solution. that said, docker support is ready for the p2pd. it's probably time to start building p2pd docker images.
